codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddEventStreamSigningDependency.java

```diff
@@ -0,0 +1,119 @@
+package software.amazon.smithy.aws.typescript.codegen;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+
+import software.amazon.smithy.codegen.core.SymbolProvider;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.model.knowledge.EventStreamIndex;
+import software.amazon.smithy.model.knowledge.TopDownIndex;
+import software.amazon.smithy.model.shapes.OperationShape;
+import software.amazon.smithy.model.shapes.ServiceShape;
+import software.amazon.smithy.typescript.codegen.LanguageTarget;
+import software.amazon.smithy.typescript.codegen.TypeScriptDependency;
+import software.amazon.smithy.typescript.codegen.TypeScriptSettings;
+import software.amazon.smithy.typescript.codegen.TypeScriptWriter;
+import software.amazon.smithy.typescript.codegen.integration.RuntimeClientPlugin;
+import software.amazon.smithy.typescript.codegen.integration.TypeScriptIntegration;
+import software.amazon.smithy.utils.ListUtils;
+import software.amazon.smithy.utils.MapUtils;
+
+public class AddEventStreamSigningDependency implements TypeScriptIntegration {
```

Comment:
Both integrations need to be added to the `META-INF/services` file for `TypeScriptIntegration` to be utilized.

Reply:
Do you mean it? https://github.com/aws/aws-sdk-js-v3/pull/1082/files#diff-c649eddc9c9bb25b274d7af1d41ad1b7R11

Reply:
Looks like I'd accidentally set something in the review filters, this is clear.